operators/hazelcast-platform-operator/5.3.0/manifests/hazelcast-platform-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,407 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "hazelcast.com/v1alpha1",
+          "kind": "Hazelcast",
+          "metadata": {
+            "name": "hazelcast"
+          },
+          "spec": {
+            "clusterSize": 3,
+            "licenseKeySecret": "hazelcast-license-key",
+            "repository": "docker.io/hazelcast/hazelcast-enterprise",
+            "version": "5.1.2"
+          }
+        },
+        {
+          "apiVersion": "hazelcast.com/v1alpha1",
+          "kind": "ManagementCenter",
+          "metadata": {
+            "name": "managementcenter"
+          },
+          "spec": {
+            "externalConnectivity": {
+              "type": "LoadBalancer"
+            },
+            "hazelcastClusters": [
+              {
+                "address": "hazelcast",
+                "name": "dev"
+              }
+            ],
+            "licenseKeySecret": "hazelcast-license-key",
+            "persistence": {
+              "enabled": true,
+              "size": "10Gi"
+            },
+            "repository": "hazelcast/management-center",
+            "version": "5.1.3"
+          }
+        },
+        {
+          "apiVersion": "hazelcast.com/v1alpha1",
+          "kind": "Map",
+          "metadata": {
+            "name": "map"
+          },
+          "spec": {
+            "hazelcastResourceName": "hazelcast"
+          }
+        }
+      ]
+    capabilities: Seamless Upgrades
+    categories: Database
+    containerImage: hazelcast/hazelcast-platform-operator:5.3
```

Comment:
```suggestion
    containerImage: docker.io/hazelcast/hazelcast-platform-operator:5.3
```

Reply:
@mvalarh Thanks 👍 
CSV file with the same container image is passed `kiwi` test and merged at https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/1305 . Does different kiwi suite run there?

Reply:
It should check also there, @J0zi can you check?

Reply:
@mvalarh both values are the same in prod whis is correct `conatainerImage` and `image`
see:

https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/1305/files#diff-69e68dc12d3292d2c4d216b016fa6f7731c14502769a193f477efe8b2b9836ebR210

https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/1305/files#diff-69e68dc12d3292d2c4d216b016fa6f7731c14502769a193f477efe8b2b9836ebR210

Reply:
@J0zi Hi, thanks for the answer. But we are not using quay.io as a registry. 
Also locally in our own environment, we scan our images using your script https://raw.githubusercontent.com/redhat-openshift-ecosystem/community-operators-pipeline/ci/latest/ci/scripts/opp.sh and tests are also passed. 

Does it mean if we add docker.io everywhere before the image name then it will pass the kiwi scan? 
As you recommended: 
`containerImage: docker.io/hazelcast/hazelcast-platform-operator:5.3`

Reply:
Hi @J0zi @mvalarh Can you please clarify what to do in our next releases? How to avoid failures on your side?